src/models/appointment.ts

```diff
@@ -0,0 +1,44 @@
+import mongoose from "mongoose";
+
+export interface IAppointment {
```

Comment:
Should also have an unique id as a string, generated by [uuid](https://www.npmjs.com/package/uuid)